content/docs/user-guide/interactivity/robotics/project-configuration.md

```diff
@@ -72,16 +71,97 @@ The ROS 2 Gem lives in the [`o3de/o3de-extras`](https://github.com/o3de/o3de-ext
 git clone https://github.com/o3de/o3de-extras
 ```
 
-### Adding Gem to your project
+### Registering the Gem
 
-To use the ROS 2 Gem in your O3DE project, you need to register the Gem with O3DE. Then, you can enable the Gem in your project. Run the following commands from the O3DE folder:
+To use the ROS 2 Gem in any O3DE project, you need to register it with O3DE.
+
+For convenience, set a couple of environment variables: `O3DE_HOME` to where your O3DE is located and `O3DE_EXTRAS_HOME`
+to the path of your cloned o3de-extras repository, for example:
+
+```shell
+export O3DE_HOME=${HOME}/o3de
+export O3DE_EXTRAS_HOME=${HOME}/o3de-extras
+```
+
+Run the following command to register the ROS 2 Gem:
+```bash
+${O3DE_HOME}/scripts/o3de.sh register --gem-path ${O3DE_EXTRAS_HOME}/Gems/ROS2
 ```
-scripts/o3de.sh register --gem-path <PATH_TO_CLONED_O3DE_EXTRAS>/Gems/ROS2
-scripts/o3de.sh enable-gem -gn ROS2 -pp <PATH_TO_YOUR_PROJECT>
+
+### Registering robotic project templates
+
+We also recommend that you register robotic project templates and their Asset Gems so that you can quickly start with your simulation project.
+If you would like to use templates, register them as well as their Asset Gems:
+
+```bash
+${O3DE_HOME}/scripts/o3de.sh register --gem-path ${O3DE_EXTRAS_HOME}/Gems/ProteusRobot
+${O3DE_HOME}/scripts/o3de.sh register --gem-path ${O3DE_EXTRAS_HOME}/Gems/RosRobotSample
+${O3DE_HOME}/scripts/o3de.sh register --gem-path ${O3DE_EXTRAS_HOME}/Gems/WarehouseAssets
+${O3DE_HOME}/scripts/o3de.sh register --gem-path ${O3DE_EXTRAS_HOME}/Gems/WarehouseSample
+${O3DE_HOME}/scripts/o3de.sh register --template-path ${O3DE_EXTRAS_HOME}/Templates/Ros2FleetRobotTemplate
+${O3DE_HOME}/scripts/o3de.sh register --template-path ${O3DE_EXTRAS_HOME}/Templates/Ros2ProjectTemplate
+```
+
+You will also want to make sure that you enable LFS (Large File Storage) which is used for Assets in `o3de-extras`:
+```bash
+cd ${O3DE_EXTRAS_HOME}
+git lfs install && git lfs pull
 ```
```

Comment:
- Nit: I think the part about "to quickly start your simulation project" makes a good introductory sentence, so moved that to the front.
- Listed the two commands as numerical steps.  I think enabling LFS makes sense as the first step

````suggestion
Robotics project templates can help you quickly start your simulation project. We recommend that you register the robotics project template Gems and their Asset Gems, which you downloaded with the `o3de-extras` repository.

To register robotic templates and assets:
1. Enable Git Large File Storage (LFS), if you haven't already.  Asset Gems use LFS to store large files. 
    ```bash
    cd ${O3DE_EXTRAS_HOME}
    git lfs install && git lfs pull
    ```
2. Register the following robotic templates and assets.
    ```bash
    ${O3DE_HOME}/scripts/o3de.sh register --gem-path ${O3DE_EXTRAS_HOME}/Gems/ProteusRobot
    ${O3DE_HOME}/scripts/o3de.sh register --gem-path ${O3DE_EXTRAS_HOME}/Gems/RosRobotSample
    ${O3DE_HOME}/scripts/o3de.sh register --gem-path ${O3DE_EXTRAS_HOME}/Gems/WarehouseAssets
    ${O3DE_HOME}/scripts/o3de.sh register --gem-path ${O3DE_EXTRAS_HOME}/Gems/WarehouseSample
    ${O3DE_HOME}/scripts/o3de.sh register --template-path ${O3DE_EXTRAS_HOME}/Templates/Ros2FleetRobotTemplate
    ${O3DE_HOME}/scripts/o3de.sh register --template-path ${O3DE_EXTRAS_HOME}/Templates/Ros2ProjectTemplate
    ```
````